packages/StackedBar/src/StackedBar.tsx

```diff
@@ -1,295 +1,294 @@
-      filter="url(#inset-shadow)"
-    />,
-    <Bar
-      key="empty-bar-3"
-      {...barConfig}
-      dataKey="thirdBar"
-      className="ids-stacked-bar__empty-bar--side"
-      radius={[0, 4, 4, 0]}
-      filter="url(#inset-shadow)"
-    />,
-  ];
-
-  const barShadow = (
-    <defs>
-      <filter id="inset-shadow">
-        <feOffset dx="0" dy="2" />
-        <feGaussianBlur stdDeviation="2" result="offset-blur" />
-        <feComposite
-          operator="out"
-          in="SourceGraphic"
-          in2="offset-blur"
-          result="inverse"
-        />
-        <feFlood floodColor="#253346" floodOpacity=".10" result="color" />
-        <feComposite operator="in" in="color" in2="inverse" result="shadow" />
-        <feComposite operator="over" in="shadow" in2="SourceGraphic" />
-      </filter>
-    </defs>
-  );
-
-  const barDataObj: Record<string, unknown> = {};
-  let bars = null;
-
-  if (dataSet && dataSet.length) {
-    bars = dataSet.map((barInfo, key) => {
-      if (barInfo.value) {
-        const dataKey = barInfo.key;
-        let pos;
-
-        if (key === 0) {
-          pos = 'first' as Pos;
+    const barConfig = {
+        stackId: "a",
+        isAnimationActive: false
+    };
+
+    const getLabel = (dataKey: string): StackedBarLabel => ({
+        dataKey,
+        formatter: (value: number) =>
+            showValue && value && size === "medium" ? value : null,
+        className: "ids-stacked-bar__label"
+    });
+
+    const getRadius = (position?: Pos): [number, number, number, number] => {
+        return [
+            position === "first" ? 4 : 0,
+            position === "last" ? 4 : 0,
+            position === "last" ? 4 : 0,
+            position === "first" ? 4 : 0
+        ];
+    };
+
+    const getColor = (dataItem: DataSet): string => {
+        if (dataItem.color) {
+            return dataItem.color;
         }
 
-        if (key === dataSet.length - 1) {
-          pos = 'last' as Pos;
+        if (dataItem.strength !== undefined) {
+            switch (dataItem.strength) {
+                case -2:
+                    return "var(--ids-stacked-bar-strength-minus-2)";
+                case -1:
+                    return "var(--ids-stacked-bar-strength-minus-1)";
+                case 1:
+                    return "var(--ids-stacked-bar-strength-1)";
+                case 2:
+                    return "var(--ids-stacked-bar-strength-2)";
+                default:
+                    return "var(--ids-stacked-bar-strength-0)";
+            }
         }
 
-        barDataObj[dataKey] = barInfo.value;
+        return "var(--ids-stacked-bar-no-color)";
+    };
 
-        return (
-          <Bar
+    const emptyBars = [
+        <Bar
+            key="empty-bar-1"
+            {...barConfig}
+            dataKey="firstBar"
+            className="ids-stacked-bar__empty-bar--side"
+            radius={[4, 0, 0, 4]}
+            filter="url(#inset-shadow)"
```

Comment:
I would use a scoped url id

Reply:
what do you mean by this? like be more specific in the name?